quantus/metrics/faithfulness/road.py

```diff
@@ -341,48 +339,53 @@ def custom_preprocess(
             A np.ndarray which contains pre-computed attributions i.e., explanations.
         s_batch: np.ndarray, optional
             A np.ndarray which contains segmentation masks that matches the input.
-        custom_batch: any
-            Gives flexibility ot the user to use for evaluation, can hold any variable.
 
         Returns
         -------
         None
         """
+
+        # Calculate accuracy for every number of most important pixels removed.
+        self.evaluation_scores = {
+            percentage: np.mean(np.array(self.evaluation_scores)[:, p_ix])
+            for p_ix, percentage in enumerate(self.percentages)
+        }
+
+    def batch_preprocess(self, data_batch: Dict[str, Any]) -> Dict[str, Any]:
+        data_batch = super().batch_preprocess(data_batch)
```

Comment:
docstring, explain this function why it is needed for what?

Reply:
done